src/Core/Customer/Domain/Entity/Customer.php

```diff
@@ -24,11 +24,13 @@ public function __construct(
     ) {
     }
 
+    /** @psalm-suppress PossiblyUnusedMethod */
```

Comment:
We are not using the suppress annotations, please check if this code is unused and if so, remove it